Dockerfile

```diff
@@ -8,6 +8,10 @@ RUN rm -rf ./*
 # Make sure the directory for individual app logs exists
 RUN mkdir -p /var/log/shiny-server
 
+# Install apt-get dependecies
+ADD apt-dependencies apt-dependencies
```

Comment:
Yes, I think this will be an empty file always present (as part of the skeleton shiny app).